.travis.yml

```diff
@@ -2,7 +2,11 @@ dist: jammy
 language: python
 
 python:
+  - "3.8"
   - "3.9"
+  - "3.10"
+  - "3.11"
+  - "3.12"
```

Comment:
Not sure why all versions weren't present here, it seems like Travis wasn't supporting some of them back then? https://github.com/izimobil/django-rest-framework-datatables/pull/139